src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -158,6 +158,7 @@ public sealed partial class SqlMetaData
         public SqlMetaData(string name, System.Data.SqlDbType dbType, long maxLength, long locale, System.Data.SqlTypes.SqlCompareOptions compareOptions, bool useServerDefault, bool isUniqueKey, System.Data.SqlClient.SortOrder columnSortOrder, int sortOrdinal) { }
         public SqlMetaData(string name, System.Data.SqlDbType dbType, string database, string owningSchema, string objectName) { }
         public SqlMetaData(string name, System.Data.SqlDbType dbType, string database, string owningSchema, string objectName, bool useServerDefault, bool isUniqueKey, System.Data.SqlClient.SortOrder columnSortOrder, int sortOrdinal) { }
+        public SqlMetaData(string name, System.Data.SqlDbType dbType, System.Type userDefinedType) { }
```

Comment:
I believe this one should also go in the `System.Data.SqlClient.NetCoreApp.cs` file. It is already a partial class so you can just add this one there and remove it from here.

Reply:
@joperezr I discussed with @saurabh500 , and it seems it can stay in `netstandard` ref